### DISCUSSION TITLE
Support .pdbrc files

### DISCUSSION THREAD

**Discussion by infinitesteps** [microsoft/debugpy#1234] [Enhancements]
I would like to use my own debug commands in the VSCode Debug Console. PDB has the option to create aliases and store them in ~/.pdbrc for all debug sessions. Does debugpy have anything like this? If not, could it be added?

For a concrete example of something I am trying to do: I would like a function foo to be available in all my debug sessions that does the following:

```python
def foo(o): # shows all non-internal methods for an object
    for m in dir(o):
        if m[0] != '_':
            print(m)
```

I can define this function in the console itself but I would like to place it (and possibly many others) in a debugger configuration file somewhere.

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.

**Comment by rchiodo** [comment]
I think this would have to be something custom. `.pdbrc` files support using pdb commands (alias being one fo them). I don't think we'd be able or want it to work that way.

But defining functions ahead of time seems doable. (We'd just eval the string for the function)
